azkaban_cli/azkaban_cli.py

```diff
@@ -537,6 +564,14 @@ def fetch_flow_execution(ctx, execution_id):
     """Fetch a flow execution"""
     __fetch_flow_execution(ctx, execution_id)
 
+@click.command(u'fetch_flow_execution_updates')
+@click.pass_context
+@click.argument(u'execution_id', type=click.STRING)
+@click.argument(u'last_update_time', type=click.STRING)
```

Comment:
Maybe set `-1` to be the default value for `last_update_time`?

Reply:
Add default value and help description to ``last_update_time``
````
 @click.option('-lt', 'last_update_time', type=click.STRING, default=u"-1", help=u'The criteria to filter by last update time', show_default=True)
````